repose-aggregator/docs/src/asciibinder/filters/openapi-validator.adoc

```diff
@@ -80,6 +83,29 @@ a| * An error occurs while attempting to read the request.
    * An unknown error occurs while validating schema from the OpenAPI document against the request.
 |===
 
+== Role Based Access Control
+For convenience an extension is provided that does role checking when used in combination with the <<keystone-v2.adoc#, Keystone v2 filter>>.
+This is enable through the extension mechanism of *Swagger/OpenApi* under the element name `x-rax-roles`.
```

Comment:
`enable` -> `enabled`.

`under the element name` -> `via the sequence named`? It seems odd to me to call it an element since we are dealing with YAML rather than XML, but I'm also not overly concerned about it.